app/routes/capture.js

```diff
@@ -27,7 +27,7 @@ module.exports = [{
     handler: async (request, h) => {
       const frn = request.payload.frn
       const captureData = await getDebts()
-      const filteredCaptureData = captureData.filter(x => x.frn === frn)
+      const filteredCaptureData = captureData.filter(x => x.frn === String(frn))
```

Comment:
Previous change had FRN validated against regex (/\d{10}/) which casts it to string, FRN is a 10 digit number so needs cast for this conmparison